eclipse-theia/theia
# #10793 Fix playwright tests for Windows and MacOS

#### What it does

- Fix workspace path encoding for Windows
- Stabilize workspace path fetching
- Fix explorer review title button selectors
- Introduce OSUtil helper functions
- Fix shortcuts for MacOS
- Improve opening of TheiaViews (speeds up tests by approx. 10s)

Contributed on behalf of STMicroelectronics

Signed-off-by: Nina Doschek <ndoschek@eclipsesource.com>

Note: I ran tests multiple times on Linux, MacOS and Windows and can confirm that tests are passing (different execution times due to different system specs).

Fixes #10793

#### How to test


Run playwright tests:
1. `yarn`
2. `yarn browser build`
3. `yarn test:playwright`
4. Confirm tests pass on different OS

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
